adr/query.py

```diff
@@ -51,6 +52,8 @@ def query_activedata(query):
                              data=query,
                              stream=True)
     response.raise_for_status()
+    if response.json().get('message') is None:
+        raise MissingDataError('ActiveData didn\'t return any data.')
```

Comment:
Issue with escaping single quotes can be bypassed if the string is formatted as:

`"ActiveData didn't return any data."`

Note the mixing of single and double quotes.